spec/requests/admin/commodities_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Api::Admin::CommoditiesController do
+  describe 'GET #index' do
+    subject(:do_request) do
+      get api_commodities_path(format: :csv)
+      response.body
+    end
+
+    before do
+      allow(TimeMachine).to receive(:at).and_call_original
+    end
+
+    it 'contains the correct headers' do
```

Comment:
We should move this test to the serializer and include some data verifications from a real row, too